modules/core/tests/server/services/influx.server.service.tests.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+require('should');
+
+var path = require('path'),
+    influxService = require(path.resolve('./modules/core/server/services/influx.server.service')),
+    config = require(path.resolve('./config/config'));
+
+describe('Service: influx', function() {
+
+  context('InfluxDB disabled', function () {
+    var originalInfluxSettings;
+
+    before(function () {
+      originalInfluxSettings = config.influxdb.enabled;
+      config.influxdb.enabled = false;
+    });
+
+    after(function () {
+      config.influxdb.enabled = originalInfluxSettings;
+    });
+
+    it('Getting client returns error if no InfluxDB configured', function(done) {
+      influxService.getClient(function(err) {
+        err.message.should.equal('No InfluxDB configured.');
```

Comment:
@simison 

i wonder what is a correct way to use `should` with asynchronous tests.

i miss calling `done(err)` [at some point](https://mochajs.org/#asynchronous-code). i.e.

```js
try {
  something.should.be.as.we.want(); // this should throw error when failed
  return done();
} catch (err) {
  return done(err);
}
```

AFAIK the purpose of assertion libraries is simply to trow error when the assertion fails.